### PR TITLE
Update all modules to have environment name

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Pårtetjåkko is a mountain in the south of the park.
 - [#1317](https://github.com/nf-core/sarek/pull/1317) - Add new tools to subway map
 - [#1325](https://github.com/nf-core/sarek/pull/1325) - Move `sentieon_dnascope_model` params into `igenomes.config`
 - [#1325](https://github.com/nf-core/sarek/pull/1325) - Refactor config files
+- [#1327](https://github.com/nf-core/sarek/pull/1325) - Update modules to have an conda environment name
```
Comment (maxulysse) [inline CHANGELOG.md:31]:
```suggestion
- [#1327](https://github.com/nf-core/sarek/pull/1327) - Update modules to have an conda environment name
```

File: modules/local/add_info_to_vcf/environment.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+name: gawk
```
Comment (maxulysse) [inline modules/local/add_info_to_vcf/environment.yml:1]: Isn't it a problem that all 3 local have the same environment name?

File: modules/nf-core/snpeff/download/environment.yml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+name: SNPEFF_DOWNLOAD
```
Comment (maxulysse) [inline modules/nf-core/snpeff/download/environment.yml:1]: why uppercase?

File: modules/nf-core/ensemblvep/download/environment.yml [hunk 4]
```diff
@@ -1,3 +1,4 @@
+name: ENSEMBLVEP_DOWNLOAD
```
Comment (maxulysse) [inline modules/nf-core/ensemblvep/download/environment.yml:1]: why uppercase?

File: modules/nf-core/sentieon/gvcftyper/environment.yml [hunk 5]
```diff
@@ -1,3 +1,4 @@
+name: SENTIEON_GVCFTYPER
```
Comment (maxulysse) [inline modules/nf-core/sentieon/gvcftyper/environment.yml:1]: why uppercase?

File: modules/nf-core/multiqc/environment.yml [hunk 6]
```diff
@@ -1,3 +1,4 @@
+name: MultiQC
```
Comment (maxulysse) [inline modules/nf-core/multiqc/environment.yml:1]: Why not fully lowercase?

**Comment by maxulysse** [comment]
cf https://github.com/nf-core/modules/pull/4332

File: modules/local/add_info_to_vcf/environment.yml (hunk 2 above)
Comment (Emiller88) [reply modules/local/add_info_to_vcf/environment.yml:1]: It _should_ just use the same conda env for all of them (just like it uses the same image, but makes a new container each process)

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
Freebayes is a bit unstable, usually a restart does it for the tests, but it's unreliable.
Remember to add untar to the exclude matrix
And sorry, but since we have updated sentieon files, all the sentieon tests are failing

**Comment by maxulysse** [comment]
> > Freebayes is a bit unstable, usually a restart does it for the tests, but it's unreliable. Remember to add untar to the exclude matrix And sorry, but since we have updated sentieon files, all the sentieon tests are failing
> 
> The sentieon-tests always fails on PR's coming from "outside nf-core", and I think that is what we are seeing here:
> 
> ```
>  Unknown config secret 'SENTIEON_AUTH_MECH_BASE64' -- Check script './workflows/../subworkflows/local/bam_variant_calling_germline_all/../bam_variant_calling_sentieon_haplotyper/../../../modules/nf-core/sentieon/haplotyper/main.nf' at line: 40
> ```

yeah, that's why I was not worried about them

**Comment by FriederikeHanssen** [comment]
yeeah will ignore the sentieon tests here. There are no crucial changes and they will be run again the moment we merge to dev

**Review by Emiller88 — APPROVED** [review]
Looks good to me! I need to wrap up the scatter stuff and bring it in here...

**Pull request merged**